docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  app:
+    container_name: review
+    restart: always
```

Comment:
restart: always can be dangerous, in case your server is misbehaving. You wouldn't really know it was messing up because it would just restarted on failure. Possibly, you would spend lots of time trying to track down the issue. 

If you have health checks written, I would use "restart: on-failure" 
Here's some info on this: https://docs.docker.com/config/containers/start-containers-automatically/

With no checks, I would set 'restart: no' or leave it out of the file as this is the default setting.